examples/usage_example.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
Personally, I love the examples. Although the documentation were really good, look a real example of program it's useful. So, I put in an independent directory. 

Unlike Java, with Python, these examples are a complete programs. Without more dependencies, compilers, or so on. So there're os useful for programmers.

If you have more examples it could be great.